elastic-stack/README.md

```diff
@@ -6,15 +6,32 @@
 
 ### DESCRIPTION
 
-Create links from SignalFx dashboards and alerts to any URL-addressable destination such as Kibana.
+Create links from SignalFx charts and alerts to any URL-addressable destination such as Kibana.
 
-The links are essentially search URLs templatize to take metric property values as search parameters.
+The links appear for metric properties for which they are targeted for (triggered for) in the configuration.
```

Comment:
delete this sentence:  The links appear for metric properties for which they are targeted for (triggered for) in the configuration.